CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+Thanks for wanting to contribute!
+
+Feel free to create issues or make pull requests, we'll try to quickly review them.
+
+If you need to reach out to us, find a relevant [issue](https://github.com/killercup/assert_cli/issues) or open a new one.
```

Comment:
I tend to write Markdown with only ref links and semantic linefeeds, but this is a short file and it doesn't really matter.